### PR TITLE
Load testing guidelines docs

### PR DESCRIPTION
**Pull request by sanchariGr**
**Proposed changes**:
- load testing guidelines docs.

### CODE REVIEW

File: docs/docs/monitoring/load-testing-guidelines.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+:::note
+
+Our team is currently in the process of running more extensive performance related tests, more information will be added here as we progress.
+
+:::
+
+## Overview
+
+In the process to gather metrics on our system's ability to handle increased loads and users we have performed tests to evaluate the maximum number of concurrent users a Rasa chatbot can handle with certain machine configurations.
+In each test case we spawned the following number of concurrent users at peak concurrency using a spawn rate of 1000 users per second. In our tests we used Rasa's [Http-Api](https://rasa.com/docs/rasa/pages/http-api),
```
Comment (ancalita) [inline docs/docs/monitoring/load-testing-guidelines.mdx:18]: Can we add a definition first of what a spawn rate is, or link to some established glossary online?

Comment (ancalita) [inline docs/docs/monitoring/load-testing-guidelines.mdx:18]:
```suggestion
In each test case we spawned the following number of concurrent users at peak concurrency using a spawn rate of 1000 users per second. In our tests we used the Rasa [HTTP-API](https://rasa.com/docs/rasa/pages/http-api),
```
I have a stylistic preference for uppercase 😄

File: docs/docs/monitoring/load-testing-guidelines.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+along with common opensource load testing tool such as [locust](https://locust.io/).
```
Comment (ancalita) [inline docs/docs/monitoring/load-testing-guidelines.mdx:19]:
```suggestion
and the [Locust](https://locust.io/). open source load testing tool.
```

File: docs/docs/monitoring/load-testing-guidelines.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+sidebar_label: Load Testing Guidelines
+title: Load Testing Guidelines
+description: |
+  Information about how best to scale up your bot to support more parallel user activity
```
[the remark this answers is not in the document]
Comment (ancalita) [reply docs/docs/monitoring/load-testing-guidelines.mdx:5]: Or we can say `to support an increased number of parallel user activity`

File: docs/docs/monitoring/load-testing-guidelines.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+## Overview
+
+In the process to gather metrics on our system's ability to handle increased loads and users we have performed tests to evaluate the maximum number of concurrent users a Rasa chatbot can handle with certain machine configurations.
+In each test case we spawned the following number of concurrent users at peak concurrency using a spawn rate of 1000 users per second. In our tests we used Rasa's [Http-Api](https://rasa.com/docs/rasa/pages/http-api),
+along with common opensource load testing tool such as [locust](https://locust.io/).
+
+|        Users             |               Configuration                        |
+|--------------------------|----------------------------------------------------|
+| Up to 50,000             |         6vCPU, 16 GB                               |
+| Up to 80,000             |         6vCPU, 16 GB but with almost 90% CPU usage |
```
Comment (ancalita) [inline docs/docs/monitoring/load-testing-guidelines.mdx:24]: Can we actually have CPU usage as a separate column actually for both cases.

**Review by ancalita — COMMENTED** [review]
Looks like a great start 💯 Please also add a doc changelog 🙏🏻

File: docs/docs/monitoring/load-testing-guidelines.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (andreaklmn) [reply docs/docs/monitoring/load-testing-guidelines.mdx:5]: +1

File: docs/docs/monitoring/load-testing-guidelines.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+sidebar_label: Load Testing Guidelines
+title: Load Testing Guidelines
+description: |
+  Information about how best to scale up your bot to support more parallel user activity
+  and how you can use tracing to help debug issues.
+---
+
+:::note
+
+Our team is currently in the process of running more extensive performance related tests, more information will be added here as we progress.
```
Comment (andreaklmn) [inline docs/docs/monitoring/load-testing-guidelines.mdx:11]:
```suggestion
Our team is currently in the process of running additional performance-related tests. More information will be added here as we progress.
```

File: docs/docs/monitoring/load-testing-guidelines.mdx (hunk 2 above)
Comment (andreaklmn) [inline docs/docs/monitoring/load-testing-guidelines.mdx:19]:
```suggestion
along with a common open source load testing tool such as [locust](https://locust.io/).
```

File: docs/docs/monitoring/load-testing-guidelines.mdx (hunk 4 above)
Comment (andreaklmn) [reply docs/docs/monitoring/load-testing-guidelines.mdx:24]: Do we only have data for these two 'user counts/reference architectures'? Or am I missing something? Perhaps we can ask CSEs to give us the most common 'reference architectures' of our customers?

**Review by andreaklmn — CHANGES_REQUESTED** [review]

File: docs/docs/monitoring/load-testing-guidelines.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (sanchariGr) [reply docs/docs/monitoring/load-testing-guidelines.mdx:5]: good catch, 'more' was a leftover from a different sentence I was trying out.

File: docs/docs/monitoring/load-testing-guidelines.mdx (hunk 4 above)
Comment (sanchariGr) [reply docs/docs/monitoring/load-testing-guidelines.mdx:24]: @ancalita I can put CPU usage as a seperate column, I stuck to the format that was used [here](https://docs.gitlab.com/ee/administration/reference_architectures/1k_users.html)

File: docs/docs/monitoring/load-testing-guidelines.mdx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+:::
```
[the remark this answers is not in the document]
Comment (sanchariGr) [reply docs/docs/monitoring/load-testing-guidelines.mdx:13]: Open to suggestions, this is to let people know that this is in progress.

File: docs/docs/monitoring/load-testing-guidelines.mdx (hunk 1 above)
Comment (sanchariGr) [reply docs/docs/monitoring/load-testing-guidelines.mdx]: Linked a locust configuration page with a definition of spawn rate mentioned but nothing specific.

File: docs/docs/monitoring/load-testing-guidelines.mdx (hunk 4 above)
Comment (andreaklmn) [reply docs/docs/monitoring/load-testing-guidelines.mdx]: okay, gotcha! but I wonder if this creates value for customers in this form if our examples are not very close to their user numbers. let's do this now and try to make it more valuable for next iteration!

**Review by andreaklmn — APPROVED** [review]

File: docs/docs/monitoring/load-testing-guidelines.mdx (hunk 4 above)
Comment (sanchariGr) [reply docs/docs/monitoring/load-testing-guidelines.mdx]: Already on it, plus hoping to get more ideas on the friday meeting with the team.

**Review by ancalita — APPROVED** [review]
Nice ⚡

**Pull request merged**